sc_wgs_monitoring/utils.py

```
@@ -67,3 +72,29 @@ def get_sample_id_from_files(files: list) -> Dict:
+        File path to the supplementary HTML file
+    pid_div_id : str
+        Id of the div to look for in the HTML file
+
+    Returns
+    -------
+    BeautifulSoup
+        BeautifulSoup object containing the HTML supplementary without the PID
+        div
+    """
+
+    with open(file) as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)